### ISSUE TITLE
Update while diashow is running

### ISSUE THREAD

**Issue by natrius** [nextcloud/gallery#538]
# Feature request

**User type**: Logged-in
**User level**: Beginners

## Description

I want to create something like live-diashow for a wedding. So i created a folder with a "just upload allowd" share. It seems like once the show is started, newly uploaded pictures are not considered anymore or not integrated in the diashow. Is it possible to add this?

## Benefit / value

It is possible to create a live-event thing. 

## Risk / caveats

Make it a setting. Otherwise maybe some questionable pictures could be possible, depending on the audience. 

## Sponsorship

Possible, yes. 

**Are you a developer willing to implement this feature?**: no
**Can you sponsor the development of this feature or do you know someone who can?**: yes/no, depends. For "my" event there is not enough time left i guess, so *shrug*

**Comment by skjnldsv** [comment]
diashow = slideshow?

**Comment by natrius** [comment]
Uhm, yes, sorry about that :D Changed it in the title as well.

**Comment by skjnldsv** [comment]
So I'm not sure I get it! :thinking: 
You want to create a slideshow that gets updated live while the folder gets more and more images?

**Comment by natrius** [comment]
Yes

**Comment by skjnldsv** [comment]
I'm not sure this is something we want. It seems like a very specific use case :thinking: @jancborchardt ?

**Comment by natrius** [comment]
Fair enough, i completely understand. It is nice for an event but other than that i don't think it is needed. Like this i have to go and check once an hour or something like that.

Was just an idea :)

**Comment by jancborchardt** [comment]
I would say it’s less about the use-case, than about the responsiveness of the web interface.

If I upload files from another device, I would also assume they directly pop up in the web interface without needing to refresh. As such I would also assume that they show up in the slideshow, as @natrius mentions.

Makes sense @skjnldsv?

**Comment by skjnldsv** [comment]
Yes, for the files list! 100%
For the slideshow it's a bit overkill, no?

**Comment by jancborchardt** [comment]
There’s plenty of cases where this would catch issues:
- You share a folder and people look through the viewer, and you delete some photos meanwhile.
- You have photos uploading from your phone via autoupload but already start showing the photos via your server.
- …

So it’s making sure the experience is smooth in all of these cases by just reflecting the current state.

**Comment by skjnldsv** [comment]
Okay! Let's keep this then, and let's open a new one on viewer! :)
We discussed this a bit today with the js team! Thanks Jan and thanks @natrius for the feedback!

**Comment by natrius** [comment]
@jancborchardt 

Well, thats exactly my usecase, to be honest. Or was that a comment to the splitted-issue?

I solved it meanwhile another way but found some suggestions for enhacements.
